app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -215,6 +211,16 @@ def finish(*_args)
 
   alias_method :abort_job, :cleanup
 
+  def cancel(*args)
+    _log.info "Job Canceling"
+    if self.state != "canceling" # ensure change of states
+      signal :cancel
+    else
+      unqueue_all_signals
+      queue_signal(:abort_job, "Job Canceled", "cancel", *args)
```

Comment:
@moolitayer I'm not sure. I agree, they should show up on the task screen. It could just be an issue with the UI filter. Are the job entries still in the table? If so, the status should be listed there.